tests/Plugin_Context_Tests.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * Tests for the Plugin_Context class.
+ *
+ * @package plugin-check
+ */
+
+use WordPress\Plugin_Check\Plugin_Context;
+
+class Plugin_Context_Tests extends WP_UnitTestCase {
+
+	/**
+	 * Create a single Plugin_Context to test.
+	 */
+	public function set_up() {
+		parent::set_up();
+
+		$this->plugin_context = new Plugin_Context( 'plugin-check/plugin-check.php' );
+	}
+
+	/**
+	 * Test class can be instantiated.
+	 *
+	 * @test
+	 */
+	public function it_can_be_instantiated() {
+		$main = new Plugin_Context( 'plugin-check/plugin-check.php' );
+
+		$this->assertIsObject( $main );
+		$this->assertInstanceOf( Plugin_Context::class, $main );
+	}
```

Comment:
See my comment on #46, I don't think we need a test for instantiation, unless that instantiation does something extra that's worth testing.